internal/nodes/avi_model_advl4_translator.go

```diff
@@ -127,7 +207,7 @@ func (o *AviObjectGraph) ConstructAdvL4PolPoolNodes(vsNode *AviVsNode, gwName, n
 		portProto := strings.Split(listener, "/") // format: protocol/port
 		// assume it to have only a single backed service, the check is in isGatewayDelete
 		svcNSName := strings.Split(svc[0], "/")
-		port, _ := strconv.Atoi(portProto[1])
+		port, _ := strconv.ParseInt(portProto[1], 10, 64)
```

Comment:
could you please use 
`utilsnet.ParsePort(port, true)` from `utilsnet "k8s.io/utils/net"`
kubernetes also uses this approach https://github.com/kubernetes/kubernetes/pull/89120/files

Reply:
Done.